mne/io/tests/test_proc_history.py

```diff
@@ -4,9 +4,9 @@
 
 import numpy as np
 import os.path as op
-from mne.io import read_info
+from mne.io import read_info, read_raw_fif
 from mne.io.constants import FIFF
-from mne.io.proc_history import _get_sss_rank
+from mne import _get_rank_sss
```

Comment:
I don't think we should put this at the root level, it seems better buried in `io.proc_history`